docs/src/main/sphinx/admin/fault-tolerant-execution.md

```diff
@@ -80,6 +80,10 @@ execution on a Trino cluster:
   - Enable compression of spooling data. Setting to `true` is recommended
     when using an [exchange manager](fte-exchange-manager).
   - ``false``
+  - `fault-tolerant-execution.exchange-encryption-enabled`
+  - Encrypt data before spooling, see [Encryption](fte-encryption) for details. 
```

Comment:
why "before" spooling ? Isnt it more like encryption and decryption during spooling.